alerts/openshift-virtualization-operator/CDIDataVolumeUnusualRestartCount.md

```diff
@@ -2,49 +2,38 @@
 
 ## Meaning
 
-This alert fires when a `DataVolume` object restarts more than three
-times.
+This alert fires when a `DataVolume` object restarts more than three times.
 
 ## Impact
 
-Data volumes are responsible for importing and creating a virtual
-machine disk on a persistent volume claim. If a data volume restarts
-more than three times, these operations are unlikely to succeed. You
-must diagnose and resolve the issue.
+Data volumes are responsible for importing and creating a virtual machine disk
+on a persistent volume claim. If a data volume restarts more than three times,
+these operations are unlikely to succeed. You must diagnose and resolve the issue.
 
 ## Diagnosis
 
-1. Obtain the name and namespace of the data volume:
+1. Find CDI pods with more than three restarts:
 
    ```bash
-   $ oc get dv -A -o json | jq -r '.items[] | \
-     select(.status.restartCount>3)' | jq '.metadata.name, .metadata.namespace'
+   $ kubectl get pods --all-namespaces -l app=containerized-data-importer -o=jsonpath='{range .items[?(@.status.containerStatuses[0].restartCount>3)]}{.metadata.name}{"/"}{.metadata.namespace}{"\n"}'
```

Comment:
Should be `oc` instead of  `kubectl`

Reply:
The 'oc' gets replaced by the script. But no problem. I can change it.